ClassTranscribeServer/Controllers/CourseOfferingsController.cs

```diff
@@ -102,11 +102,6 @@ public async Task<ActionResult<List<CourseOffering>>> DeleteCourseOffering(strin
             return courseOfferings;
         }
 
-        private bool CourseOfferingExists(string id)
```

Comment:
I'd make the id 'courseId' so it is obvious that it is not courseoffering Id, and document this.

Reply:
This function is being removed in this PR. Do you want to keep it?

I removed it because it's not being used at all right now.

Reply:
Haha. No it can go.